plugins/out_pgsql/pgsql.c

```diff
@@ -54,6 +54,7 @@ static int cb_pgsql_init(struct flb_output_instance *ins,
     char *temp = NULL;
     const char *tmp = NULL;
     int ret;
+    int daemon;
```

Comment:
Probably a good name change will be `daemon_mode` since its a boolean and makes it clear it's related to run in a specific mode called `daemon`

Reply:
The reason I choose "daemon" as a name for background-mode of this plugin is that in SERVICE SECTION, there is a similar configurable value with similar function called "deamon". So I refer to this
![image](https://user-images.githubusercontent.com/46041550/234739768-1e137ca8-2a5d-4338-945b-dc903312af48.png)
If you insist "daemon_mode" a better name, I will update it.
Thanks for your kind review.

Reply:
@TomlinfreeGit yes, that refers to the main configuration, it's ok to me to use daemon or daemon_mode, the only think it's to make it clear in the documentation so people don't get confused